### PR TITLE
Release 1.1.2.0

### PR DESCRIPTION
**Pull request by poorlyknitsweater**
I think we don't need to change anything but this, but you could check if Hackage accepts the version bounds (I used `cabal gen-bounds` and widened them to the next major versions).

### CODE REVIEW

**Review by fatho — APPROVED** [review]
LGTM (I did `stack sdist`, and then successfully built that source tarball elsewhere using cabal)! Can you also create a release on GitHub after merging?

Then I'll upload it to hackage from there.

**Comment by poorlyknitsweater** [comment]
@OpsBotPrime help

**Comment by OpsBotPrime** [comment]
That was not a valid command.

**Comment by poorlyknitsweater** [comment]
@OpsBotPrime merge

**Comment by OpsBotPrime** [comment]
Pull request approved for merge by @poorlyknitsweater, rebasing now.

**Comment by OpsBotPrime** [comment]
Rebased as a5a71a7d74028d7d993e9e12662ec493cbe4b064, waiting for CI …

**Pull request merged**
